packages/coinstac-api-server/src/auth-helpers.js

```diff
@@ -205,6 +245,64 @@ const helperFunctions = {
       res(Boom.unauthorized('Incorrect username or password.'));
     }
   },
+  /**
+   * Confirms that submitted email is valid
+   * @param {object} req request
+   * @param {object} res response
+   * @return {object} The requested object
+   */
+  async validateEmail(req, res) {
```

Comment:
I'd suggest here to not create this function. You could use the already existing `validateUniqueEmail` function, which does the same thing. Maybe rename it if you want.